build-local.sh

```diff
@@ -0,0 +1 @@
+docker build -t ghcr.io/celestiaorg/hyperlane-init:local -f hyperlane/Dockerfile .
```

Comment:
If you want you can just add this to the makefile as:

`make docker-build-hyperlane`